object-search/tests/test_objectsearch.py

```diff
@@ -0,0 +1,110 @@
+import base64
+import io
+import json
+import os
+import sys
+import shutil
+import subprocess
+
+from PIL import Image
+import pytest
+
+from jina.flow import Flow
+
+
+os.chdir(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..' ))
```

Comment:
why do we need to change dir and not to pass `paths` relative to the file dir?

Reply:
The index and query flow reference pod yml paths and so we need to change the path to root of example.

Reply:
is it that so? are we sure? it feels like a smell (I am not sure)